src/lib/Scenes/Partner/Components/PartnerShows.tsx

```diff
@@ -114,10 +120,17 @@ export const PartnerShows: React.FC<{
     [partner.pastShows.edges, hasRecentShows, pastShows]
   )
 
+  const tabContext = useContext(StickyTabPageFlatListContext)
+
+  const tabIsActive = Boolean(useNativeValue(Animated.eq(tabContext.activeTabIndex, tabContext.tabIndex), 0))
```

Comment:
might make sense to expose this tabIsActive value in the context rather than the index values. Not sure how useful they are in any other situation.